ckanfunctionaltests/api/test_search_datasets.py

```diff
@@ -8,8 +8,11 @@
 from ckanfunctionaltests.api.conftest import clean_unstable_elements
 
 
-def _get_limit_offset_params(base_url):
-    return ("rows", "start",) if base_url.endswith("/3") else ("limit", "offset",)
+def _get_limit_offset_params(variables, base_url):
+    if variables.get("ckan_version") == "2.9":
+        return ("rows", "start",)
+    else:
+        return ("rows", "start",) if base_url.endswith("/3") else ("limit", "offset",)
```

Comment:
A bit worrying that we're seeing duplicated `== "2.9"` and `endswith("/3")` conditionals - can we centralise this logic in one place so that this forking doesn't spread throughout the codebase?

If it were me, I'd encapsulate this info in a class, so that it can be called like `SearchDecorator.new(variables, base_url).limit_offset_params` 🤔

Reply:
I'll see what I can do

Reply:
There's only one `endswith("/3")` conditional in the codebase, I've however updated the fixture to not duplicate the return